packages/dx-react-grid-demos/src/demo-sources/grid-editing/bootstrap4/inline-cell-editing.jsx

```diff
@@ -0,0 +1,137 @@
+import React, { useState } from 'react';
+import Paper from '@material-ui/core/Paper';
+import { EditingState } from '@devexpress/dx-react-grid';
+import {
+  Grid,
+  Table,
+  TableHeaderRow,
+  TableInlineCellEditing,
+  Toolbar,
+} from '@devexpress/dx-react-grid-bootstrap4';
+import {
+  Plugin,
+  Template,
+  TemplatePlaceholder,
+} from '@devexpress/dx-react-core';
+
+import {
+  generateRows,
+  defaultColumnValues,
+} from '../../../demo-data/generator';
+
+const getRowId = row => row.id;
+
+// #FOLD_BLOCK
+const StartEditActionSelector = (props) => {
```

Comment:
Can we write only one `jsxt` demo and add theme partial parts? Like we have done here - [scatter.partial](https://github.com/DevExpress/devextreme-reactive/tree/master/packages/dx-react-chart-demos/src/demo-sources/zoom-pan)